keyboards/ymdk/ymd09/keymaps/andys8/keymap.c

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 andys8 <andys8@users.noreply.github.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+#include "sendstring_german.h"
+
+enum custom_keycodes {
+    GIT_STASH = SAFE_RANGE,
+    GIT_STASH_POP,
+    GIT_COMMIT,
+    COPY_PASTE,
+    E_INFO,
+    E_WARN,
+    E_TABLE,
+    E_SHRUG
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case E_INFO:
+            if (record->event.pressed) {
+                send_unicode_string("ℹ️");
+            }
+            break;
+        case E_WARN:
+            if (record->event.pressed) {
+                send_unicode_string("⚠️");
+            }
+            break;
+        case E_TABLE:
+            if (record->event.pressed) {
+                send_unicode_string("(╯°□°)╯︵ ┻━┻");
+            }
+            break;
+        case E_SHRUG:
+            if (record->event.pressed) {
+                send_unicode_string("¯\\_(ツ)_/¯");
+            }
+            break;
+        case GIT_STASH:
+            if (record->event.pressed) {
+                send_string("git stash"SS_TAP(X_ENTER));
+            }
+            break;
+        case GIT_STASH_POP:
+            if (record->event.pressed) {
+                send_string("git stash pop"SS_TAP(X_ENTER));
+            }
+            break;
+        case GIT_COMMIT:
+            if (record->event.pressed) {
+                send_string("git add -A && git commit -a"SS_TAP(X_ENTER));
+            }
+            break;
+        case COPY_PASTE:
+            if (record->event.pressed) {
+                send_string(SS_LCTL("c"));
+            } else {
+                send_string(SS_LCTL("v"));
```

Comment:
```suggestion
                tap_code16(C(KC_C));
            } else {
                tap_code16(C(KC_V));
```

Reply:
This one is also an improvement. Thanks ✅